libmproxy/flow_export.py

```diff
@@ -0,0 +1,70 @@
+import urllib
+import netlib.http
+
+
+def curl_command(flow):
+    data = "curl "
+
+    for k, v in flow.request.headers.fields:
+        data += "-H '%s:%s' " % (k, v)
+
+    if flow.request.method != "GET":
+        data += "-X %s " % flow.request.method
+
+    full_url = flow.request.scheme + "://" + flow.request.host + flow.request.path
+    data += "'%s'" % full_url
+
+    if flow.request.content:
+        data += " --data-binary '%s'" % flow.request.content
+
+    return data
+
+
+def python_code(flow):
+    code = """import requests
+
+url = '{url}'
+{headers}{params}{data}
+response = requests.request(
+    method='{method}',
+    url=url,{args}
+)
+
+print(response.text)"""
```

Comment:
Can you use [textwrap.dedent](https://docs.python.org/2/library/textwrap.html#textwrap.dedent) here to make it more readable?